arches/app/templates/index.htm

```diff
@@ -90,26 +90,43 @@ <h1>{% blocktrans %}Arches | {{version}}{% endblocktrans %}</h1>
                             </li>
                         {% else %}
                             <li>
-                                <a class="auth-welcome" href="{% url 'user_profile_manager' %}">
-                                    {% trans "Welcome, " %} {{ user.first_name|default:user.username }}
-                                </a>
+                                <a href="#info-block-1">{% trans "Arches Features" %}</a>
```

Comment:
The way that you've reworked this here breaks this template, as it leaves the above conditional without an `endif` closing tag.

Reply:
Looks like some merging issues happened here, now fixed to be in-line with dev/6.0.x